### PR TITLE
bump YouTube patches to v17.32.38

### PR DESCRIPTION
**Pull request by johnconner122**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,35 +66,35 @@ Official patches by ReVanced
```
Comment (Canny1913) [inline README.md:66]: This file shouldn't be changed

**Comment by lbchX** [comment]
Good Evening to whole team!
I probably found a bug which has to do with "playing again" video after 
its duration stops...

**Comment by TheJeterLP** [comment]
> Good Evening to whole team!
> I probably found a bug which has to do with "playing again" video after 
> its duration stops...
> 

No. Disable auto-repeat in settings.

**Comment by lbchX** [comment]
> > Good Evening to whole team!
> > I probably found a bug which has to do with "playing again" video after
> > its duration stops...
> 
> No. Disable auto-repeat in settings.

Unfortunately nothing happens (to me) even with "disabled autplaying"...

**Comment by TheJeterLP** [comment]
> > > Good Evening to whole team!
> > > I probably found a bug which has to do with "playing again" video after
> > > its duration stops...
> > 
> > No. Disable auto-repeat in settings.
> 
> Unfortunately nothing happens (to me) even with "disabled autoplaying"...

For me it does not always repeat. Anyways, create a proper issue instead of commenting here.

**Review by GeekCornerGH — APPROVED** [review]
Tried and working fine

**Comment by E85Addict** [comment]
17.32.39 is as well. It's what im using.

**Comment by oSumAtrIX** [comment]
> Tried and working fine

Have you checked all settings? Such as wide search bar or tablet mini-player?

**Comment by oSumAtrIX** [comment]
Closing in favour of #398.

**Pull request closed without merging**